### PR TITLE
Content - Feedback - Update documentation

### PR DESCRIPTION
**Pull request by duboisp**
* Apply updates to implementation as discussed with DTO during office hours 
* Improved the code sample displayed in the working example
* Fix the grunt script to not rely on a generated file when initiating any task
* Updated the generated content

### CODE REVIEW

**Comment by duboisp** [comment]
@LanaStewa FYI

File: sites/feedback/index.json-ld [hunk 1]
```diff
@@ -450,23 +456,53 @@
...
+					"You can configure the page feedback tool via the page properties by specifying the optional configuration (contact info and section). The value for the theme configuration are automatically set by the MWS.",
+					"For additional guidance on how to implements, you can consult the guide on GCPedia along with the other documentation for the managed web service."
+				],
+				"fr": [
+					"Veuillez lire les <a href=\"https://conception.canada.ca/retroaction/index.html\">consignes du système de conception</a> avant d'ajouter l'outil de rétroaction sur la page à vos pages.",
+					"Si vous configurez la section, veuillez suivre les <a href=\"https://conception.canada.ca/retroaction/index.html\">consignes supplémentaires du système de conception</a>.",
+					"Vous pouvez configurer l'outil de rétroaction sur la page via les propriétés de la page en spécifiant la configuration facultative (informations de contact et section). Les valeurs de la configuration du thème sont automatiquement définies par le SWG.",
+					"Pour des conseils supplémentaires sur la mise en œuvre, vous pouvez consulter le guide sur GCPedia avec les autres documents du service web géré."
```
Comment (Garneauma) [inline sites/feedback/index.json-ld:481]:
```suggestion
					"Pour des conseils supplémentaires sur la mise en œuvre de l'Outil de rétroaction sur la page, vous pouvez consulter le guide sur GCPedia avec les autres documents du SWG."
```

File: sites/feedback/index.json-ld [hunk 2]
```diff
@@ -450,23 +456,53 @@
...
 			"name": {
-				"en": "Standard",
-				"fr": "Standard"
+				"en": "Standard (wet-boew)",
+				"fr": "Standard (wet-boew)"
 			},
 			"introduction": {
-				"en": "This implementation is meant for developers/publishers adding the component manually.",
-				"fr": "Cette implémentation est destinée aux développeurs/éditeurs qui ajoutent le composant manuellement."
+				"en": "This implementation is meant for developers/publishers adding the component manually with using the latest GCWeb version along with at least the implementation of the page details version 2.x.",
```
Comment (Garneauma) [inline sites/feedback/index.json-ld:493]:
```suggestion
				"en": "This implementation is meant for developers/publishers adding the component manually while using the latest version of GCWeb along with the implementation of the page details version 2.0 and above.",
```

File: sites/feedback/index.json-ld [hunk 3]
```diff
@@ -450,23 +456,53 @@
...
+				"fr": "Cette implémentation est destinée aux développeurs/éditeurs qui ajoutent le composant manuellement en utilisant une version récente de GCWeb avec l'implémentation de la version 2.x du détails de la page."
```
Comment (Garneauma) [inline sites/feedback/index.json-ld:494]:
```suggestion
				"fr": "Cette implémentation est destinée aux développeurs/éditeurs qui ajoutent le composant manuellement en utilisant la dernière version de GCWeb avec l'implémentation de la version 2.0 ou plus du détails de la page."
```

File: sites/feedback/index.json-ld [hunk 4]
```diff
@@ -450,23 +456,53 @@
...
+					"Ensure the latest version of <a href=\"../page-details/page-details-docs-en.html\">page details</a> is implemented",
 					"To show the page feedback form, create a <code>&lt;div class=\"wb-disable-allow\"&gt;</code> with the following data attributes:<ul><li>Mandatory: <code>data-ajax-replace=\"assets/page-feedback-en.html\"</code></li><li>Mandatory: <code>data-feedback-link=\"[Contact link text]\"</code></li><li>Mandatory: <code>data-feedback-url=\"[Contact link URL]\"</code></li><li>Optional: <code>data-feedback-theme=\"[Text defining the theme of your page]\"</code></li><li>Optional: <code>data-feedback-section=\"[Text defining the section where your page resides]\"</code></li></ul>",
 					"Additionally, your page needs to have the following metadata: <code>&lt;meta name=\"dcterms.creator\" content=\"[Department name]\"&gt;</code>"
 				],
 				"fr": [
+					"Vérifier qu'une version récente de GCWeb (<a href=\"https://github.com/wet-boew/GCWeb/releases\">v13.9.0 ou plus récent</a>) est implémenté",
+					"Vérifier que la <a href=\"../page-details/page-details-docs-en.html\">version 2.1, ou plus récente, du détails de la page</a> est implémenté",
```
Comment (Garneauma) [inline sites/feedback/index.json-ld:505]:
```suggestion
					"Vérifier que la <a href=\"../page-details/page-details-docs-en.html\">version 2.1, ou plus récente, du détails de la page</a> est implémentée.",
```

**Review by Garneauma — CHANGES_REQUESTED** [review]
- [x] Quelques changements de contenu.
- [ ] Pourrais-tu également modifier l'URL du ajax-replace pour: "https://canada.ca/etc/designs/canada/wet-boew/assets/feedback/page-feedback-en.html".

**Review by Garneauma — APPROVED** [review]

**Pull request merged**